src/RepoToolset/DefaultVersions.props

```diff
@@ -1,30 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
+
   <!--
     Sets versions of packages used by the toolset unless they were already specified by the repo.
   -->
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
```

Comment:
I'd rather not add this everywhere. The toolset files won't change in the repo unless the repo restores a new version of the package.

Reply:
This is best practice to have, regardless of whether the files should change.